commands/core/uninstall.go

```diff
@@ -26,9 +26,20 @@ import (
 
 // PlatformUninstall FIXMEDOC
 func PlatformUninstall(ctx context.Context, req *rpc.PlatformUninstallRequest, taskCB rpc.TaskProgressCB) (*rpc.PlatformUninstallResponse, error) {
+	if err := platformUninstall(ctx, req, taskCB); err != nil {
+		return nil, err
+	}
+	if err := commands.Init(&rpc.InitRequest{Instance: req.Instance}, nil); err != nil {
+		return nil, err
+	}
+	return &rpc.PlatformUninstallResponse{}, nil
+}
+
+// platformUninstall is the implementation of platform unistaller
+func platformUninstall(ctx context.Context, req *rpc.PlatformUninstallRequest, taskCB rpc.TaskProgressCB) error {
```

Comment:
use a nested function, this way we could remove parameter duplication and could use the defer statement